CDCgov/prime-reportstream
# Engagement/angela/8644 soap filename null

This PR ...

Test Steps:
I'm not sure how to properly test this, but I ran the SoapTransportIntegrationTests on debug and was able to see that a report id is accessible at that point whereas filename was null. These don't seem to actually go to the database, though.

## Changes
- Changed where it uses "filename" to use "report id" when logging a successful SOAP transport to be consistent with the logging of failed transports and make it easier to track what happened with a report.

### Testing
- [ ] Tested locally? -Sort of, as stated in Test Steps

### Process

## Linked Issues
- Fixes #8644
